101-webapp-privateendpoint-vnet-injection/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {

```

Comment:
remove these when you add the defaultValues

Reply:
why do you prefer default values, I prefer parameters, it's more explicit

Reply:
defaultValues make it easier for users to deploy the template and understand possible values

Reply:
ok, done

Reply:
ok I understand, done, I added default values for each parameter